provider/inspec.rb

```diff
@@ -132,7 +133,34 @@ def format_url(url)
     def generate_resource_tests(data)
       target_folder = File.join(data[:output_folder], 'test')
       FileUtils.mkpath target_folder
+
       FileUtils.cp_r 'templates/inspec/tests/.', target_folder
+
+      name = "google_#{data[:product_name]}_#{data[:object].name.underscore}"
+
+      generate_resource_file data.clone.merge(
```

Comment:
nit: can you move this generate_resource_file code to a different function to avoid some of the duplication? I think it'll make it easier to understand what's going on too.